src/sections/Community/Handbook/Handbook.style.js

```diff
@@ -35,7 +35,11 @@ export const HandbookWrapper = styled.div`
       color: ${(props) => props.theme.tertiaryColor};
       transition: 0.8s cubic-bezier(0.2, 0.8, 0.2, 1);
     }
-
+    #less-padding {
```

Comment:
@kituuu Please don't use id in syles.

Reply:
Okay, can you suggest a better approach. Like by changing class properties. I can fix the 4-5 pages mentioned in the issue but other page got affected. I thought by adding a id to only those elements which needs to be fixed is a better plan because it won't affect other elements/components utilizing the same css class name. @Chadha93

Reply:
I fix this : ASAP